Lib/test/test_minidom.py

```diff
@@ -1560,5 +1561,21 @@ def testProcessingInstructionNameError(self):
         pi = doc.createProcessingInstruction("y", "z")
         pi.nodeValue = "crash"
 
+    def test_minidom_attribute_order(self):
+        doc = Document()
+        root = doc.createElement("curriculum")
+        root.setAttribute( "status", 'public' )
+        root.setAttribute( "company", 'example' )
+        doc.appendChild(root)
+
+        with open(support.TESTFN, "w") as xml_file:
```

Comment:
Don't use `open()` in text mode without explicit encoding in tests (unless you test `open()` itself). The result is environment depending. It is better to use `io.StringIO` in this test.

Reply:
You are right, done. 

I just saw your comment in the @rhettinger PR saying the same